apps/marketplace/components/BuyerTraining/BuyerTrainingAdditionalInformationStage.scss

```diff
@@ -1,13 +1,19 @@
-.closingDateControl {
+.noticeBar {
   margin-top: 1.5rem;
+  margin-bottom: 1.5rem;
+  border-left: 10px solid #00698F;
+  background-color: #daf3fe;
+  color: #00698F;
 }
 
-.closingStageContainer {
-  p {
+.additionalInformationContainer {
```

Comment:
can some of the generic styles be used for this?

Reply:
resolved